.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 script:
   # Travis aborts a job if it doesn't write to console for more than 10 minutes, use travis_wait to
   # Travis provides us with a travis user and travis database
-  - travis_wait 30 mvn verify --batch-mode --quiet -Dmaven.test.redirectTestOutputToFile=true -Ddb_user=travis -Ddb_password -Dit_db_name=travis -P!create-it-db
+  - travis_wait 50 mvn verify --batch-mode --quiet -Dmaven.test.redirectTestOutputToFile=true -Ddb_user=travis -Ddb_password -Dit_db_name=travis -P!create-it-db
```

Comment:
This is awkward, why do we suddenly need significantly more time to run the integration tests?